packages/block-library/src/gallery/style.scss

```diff
@@ -101,11 +91,11 @@
 			margin-top: 0;
 			margin-bottom: auto;
 			img {
-				margin-bottom: var(--gallery-block--gutter-size, #{$grid-unit-20});
+				margin-bottom: var(--wp--style--unstable-gallery-gap, #{$grid-unit-20});
 			}
 
 			figcaption {
-				bottom: var(--gallery-block--gutter-size, #{$grid-unit-20});
+				bottom: var(--wp--style--unstable-gallery-gap, #{$grid-unit-20});
```

Comment:
should this be `bottom` or `margin-bottom`?

Reply:
It was using `bottom` to correctly align the caption when the images were not cropped, but it seems like it is not needed with the move to using gap instead of margin, so have removed this.